rackcheck/info.rkt

```diff
@@ -3,7 +3,8 @@
 (define version "2.0.1")
```

Comment:
You can drop the `version` here.

Reply:
Likewise, I think you should keep the `examples` folder in the root of the repository and then you can drop the `{compile,test}-omit-paths` fields. I don't think there's any need to distribute the examples outside of the repo.

Reply:
Examples restored to root of the repo in: https://github.com/Bogdanp/rackcheck/pull/10/commits/f104664d7f08e0091b06440a920cd12ea8c76725 https://github.com/Bogdanp/rackcheck/pull/10/commits/c092edebb0cc58b9a12038c18cd9fb19632046c7 https://github.com/Bogdanp/rackcheck/pull/10/commits/e9ddaff9888b55be0252c7276eab0a830be95515 https://github.com/Bogdanp/rackcheck/pull/10/commits/670969362ef7164708568e4c26a6efe01de1a178 https://github.com/Bogdanp/rackcheck/pull/10/commits/5d77dc5139204be3d74e78a3bbbb514a28701cf5